src/ui/RadarMap.ts

```diff
@@ -155,6 +155,15 @@ class RadarMap extends maplibregl.Map {
     }
     this.on('styledata', onStyleLoad);
   }
+
+  fitToMarkers(fitBoundsOptions: maplibregl.FitBoundsOptions = { padding: 50 }, overrideMarkers?: RadarMarker[]) {
+    const bounds = new maplibregl.LngLatBounds();
+    const markers = overrideMarkers || this._markers;
+    markers.forEach(marker => {
```

Comment:
We should probably add a lint rule for this, but there should be parens around the arguments
```
markers.forEach((markers) => {
```